index.html

```diff
@@ -81,132 +72,28 @@
+        
+        <p>In addition, language tags are also used to identify <a>international preferences</a> linked to culture, region, or language. Because these preferences are linked to the natural language or regional association of the end user, [[BCP47]] provides a natural way of encoding these preferences also. Such preferences are applied to processes such as presenting numbers, dates, or times; sorting lists linguistically; providing defaults for items such as the presentation of a calendar, common units of measurement, or 12- vs. 24-hour time presentation; and many other details that users might find too tedious to set individually. Collectively, these preferences are usually called a <a>locale</a>.</p>
+        
+        <p>This document describes best practices for the adoption and use of [[BCP47]] language tags for the identification of <a>natural language</a> content as well as the use of language tags to represent the locale preferences of the user or content author. It describes how document formats, specifications, and implementations should handle the language tags described by [[BCP47]], as well as data structures that extend these tags to describe <a>international preferences</a> (see also sec. 3.1 in [[WS-I18N-SCENARIOS]]). </p>
+        
+        <p>Identification of language and locale has a broad range of applications within the World Wide Web. Existing standards which make use of language identification include the <code>xml:lang</code> attribute in [[XML10]], the <code>lang</code> and <code>hreflang</code> atttributes in [[HTML]], the <code>language</code> property in [[XSL10]], and the <code>:lang</code> pseudo-class in CSS [[CSS3-SELECTORS]]. Language tags are also used to identify locales, such as in the Unicode Common Locale Data Repository or "CLDR" project [[CLDR]].</p>
+        
+        <p>Historically, locales were identified by the programming language or operating environment of the user. This application-specific identifier was often inferred from language tags. For example, an implementation could map a language tag from an existing protocol, such as HTTP's Accept-Language header, to its locale model. Locales may also be identified directly by using the language tag syntax in data items (elements, attributes, headers, etc.) that explicitly serve the purpose of locale identification.</p>
+        
+        <p>Unicode Locales are an extension of [[BCP47]] defined as part of Unicode's [[CLDR]] project. These identifiers are identical to language tags, but apply additional rules about the content of certain language tags. Unicode Locales increasingly form the basis for <a>internationalization</a> on the Web, particularly as part of the <code>Intl</code> locale framework in JavaScript.</p>
```

Comment:
We should probably add a link to the definition of "Unicode Locales" here (i.e., `<a>Unicode Locales</a>`). And it is "Unicode locales" (lowercase) in the definition in section 2.

Reply:
Fixed in 645ee75